requirements.txt

```diff
@@ -1,7 +1,13 @@
 appdirs==1.4.3
+
 attrs==18.2.0
 black==18.9b0
+
+
 Click==7.0
+
 Django==2.1.4
+
 pytz==2018.7
+
```

Comment:
Can we remove the spaces here? As we don't actually edit a `requirements.txt` manually, we update it on the terminal.

Reply:
oh that i will do it